core/trino-main/src/main/java/io/trino/testing/LocalQueryRunner.java

```diff
@@ -1079,5 +1137,20 @@ public LocalQueryRunner build()
                     planOptimizersProvider,
                     operatorFactories);
         }
+
+        public LocalQueryRunner buildWithSessionManager()
```

Comment:
just add a method:
```
withSystemSessionProperties(Set<SystemSessionPropertiesProvider> systemSessionProperties)
```

Reply:
Added extra method `getSystemSessionProperties` to provide new `SystemSessionPropertiesProvider` with latest `featuresConfig` reference if no `Set<SystemSessionPropertiesProvider>` is set.